resources/install.rb

```diff
@@ -223,11 +225,20 @@ def hab_path
   end
 
   def hab_command
-    cmd = if node['kernel']['release'].to_i < 3
-            ["bash #{Chef::Config[:file_cache_path]}/hab-install.sh", "-v #{hab_version} -t x86_64-linux-kernel2"]
-          else
-            ["bash #{Chef::Config[:file_cache_path]}/hab-install.sh", "-v #{hab_version}"]
-          end
+    if new_resource.hab_version
```

Comment:
this whole block feels a bit wrong to me, would the following not be better/simpler?

```
cmd_tmp = "bash #{Chef::Config[:file_cache_path]}/hab-install.sh"
cmd_tmp << " -v #{new_resource.hab_version} " if new_resource.hab_version
cmd_tmp << " -t x86_64-linux-kernel2" if node['kernel']['release'].to_i < 3
cmd = [cmd_tmp]
```

Reply:
so, that's how it ended up after some comments from @tas50 I'm happy to make changes to it if you have some suggestions

Reply:
it doesn't like your `cmd = [cmd_tmp]`

Reply:
had to tweak this a bit but, working and tested